src/Vulkan/RHI/VkRHI.cpp

```diff
@@ -386,6 +387,11 @@ void VkRHI::ModifyShaderCompilerEnvironment(ShaderCompilerBackend compilerBacken
         shaderEnv.args.emplace_back(L"-fvk-bind-resource-heap");
         shaderEnv.args.emplace_back(L"0");
         shaderEnv.args.emplace_back(L"1");
+        shaderEnv.args.emplace_back(L"-fvk-use-scalar-layout");
+    }
+    else if (compilerBackend == ShaderCompilerBackend::Slang)
+    {
+        shaderEnv.args.push_back(L"--scalar-block-layout");
```

Comment:
Where'd you find this one? Looking at slang docs I wasn't able to find this env arg